tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Think the target needs to be set to es6 so that the `esModuleInterop` flag works as expected, no?

Reply:
It's working on my local with es5.

Reply:
It's the typescript tests that are failing. Just trying to get some tests working for the OAI-generator tests to work properly.